src/apiCall.js

```diff
@@ -1,7 +1,11 @@
 const getQuote = () => {
     return fetch('https://api.kanye.rest/')
     .then(response => checkResponse(response))
```

Comment:
Be sure to add some error handling around these fetch calls in case something goes wrong with the request (thinking around at least checking is `response.ok` is true).